client/login/magic-login/index.jsx

```diff
@@ -1397,7 +1397,7 @@ const mapState = ( state ) => ( {
 	isSendingEmail: isFetchingMagicLoginEmail( state ),
 	emailRequested: isMagicLoginEmailRequested( state ),
 	emailRequestError: getMagicLoginRequestEmailError( state ),
-	isJetpackLogin: getCurrentRoute( state ) === '/log-in/jetpack/link',
+	isJetpackLogin: getCurrentRoute( state )?.startsWith( '/log-in/jetpack/link' ),
```

Comment:
It could simply be `.includes('/jetpack')`, isn't it?

Reply:
I think it could, I just wanted to keep the same to avoid any unwanted changes in other places. Do you think it's safe to change?

Reply:
It's fine. Let us leave it as is.